src/crypto/eckey.h

```diff
@@ -62,8 +70,11 @@ struct wickr_ec_curve {
 };
 
 typedef struct wickr_ec_curve wickr_ec_curve_t;
+//TODO(Tom): Decide if key lengths should be part of the struct
```

Comment:
no key length required unless we need them for something specific?

Reply:
My main question is, where is a good place to store key lengths. I thought they could be attached to the corresponding curves